unstructured/staging/base.py

```diff
@@ -39,6 +39,35 @@ def _get_metadata_table_fieldnames():
 ] + _get_metadata_table_fieldnames()
 
 
+def convert_to_text(elements: List[Element]) -> str:
+    """Converts a list of elements into clean, concatenated text."""
+    clean_concat_text: str = ""
```

Comment:
question for reviewer: will adding all the text to one string like this cause a problem for very large files?